### ISSUE TITLE
Editor field close button doesn't close editor.

### ISSUE THREAD

**Issue by enginne** [themeum/kirki#1584]
### Issue description:
As the title says the close button of the editor field doesn't close the editor, thus the only option is to close the customizer panel. Everything else about the editor works fine.
Seems like a js bug in the button.

### Version used:
Using latest Version 3.0.11, although i had the same problem with Version 3.0.10

### Using theme_mods or options?
theme_mods

### Code to reproduce the issue (config + field(s))

```
add_action( 'after_setup_theme', 's_tests_fields' );
function s_tests_fields()
{

	Kirki::add_field( 'my_config', array(
		'type'			=> 'editor',
		'settings'		=> 'block-about-content',
		'description'	=> __( 'Write your bio or other content for this section.', 'andy' ),
		'section'		=> 'tests',
		'default'		=> 'Default text',
		'priority'		=> 10,
		'sanitize_callback'	=> 'wp_kses_post',
	));
}
```
Tests using your _s theme and a WP fresh install.

**Comment by lucas-viewup** [comment]
@enginne, I discovered a shortfix for it, but @aristath should be aware of it, I tested and it worked.
**https://github.com/aristath/kirki/blob/develop/controls/editor/editor.js:76**
```
        /**
	 * Toggle the editor.
	 */
	toggleEditor: function() {

		var control = this,
		    editorWrapper = jQuery( '#kirki_editor_pane' );
		
		var isHidden = editorWrapper.hasClass('hide');

		editorWrapper.removeClass();
		if ( isHidden ) {
			editorWrapper.addClass( control.id );
		} else {
			editorWrapper.addClass( 'hide' );
		}
	},
```
Let me know when you need the PR.
Regards,

**Comment by aristath** [comment]
fixed in #1596

**Comment by aristath** [comment]
v3.0.12 refactored the editor field and now implements the WP-Core control which is added in WordPress 4.9 (to be released in a few days).
